File: soutLink-frontend/src/app/services/models/upcoming-appointments-response.ts
/* tslint:disable */
/* eslint-disable */
import { StudentResponse } from '../models/student-response';
export interface UpcomingAppointmentsResponse {
  appointmentDate?: string;
  appointmentTime?: string;
  description?: string;
  id?: number;
  objet?: string;
  student?: Array<StudentResponse>;
}

File: soutLink-backend/src/main/java/com/soutLink/supervisor/dto/UpcomingAppointmentsResponse.java
package com.soutLink.supervisor.dto;

import com.soutLink.enums.AppointmentStatus;
import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UpcomingAppointmentsResponse {
    private Long id;
    private String description;
    private String objet;
    private List<StudentResponse> student;
    private String appointmentDate;
    private String appointmentTime;
}
